### ISSUE TITLE
use same esm version for dependencies 

### ISSUE THREAD

**Issue by schlichtanders** [esm-dev/esm.sh#609]
## Failing module

when accessing https://esm.sh/v64/@mui/material@5.11.15?alias=react:preact/compat,react-dom:preact/compat,react/jsx-runtime:preact/jsx-runtime&deps=preact@10.6.6&target=es2020 and recursively looking into the imports, you will find that the preact is imported from `esm.sh/stable` instead of `esm.sh/v64`

This is crucial, as esm.sh unfortunately introduces new bugs (I have one right which I cannot minify yet but it is not occuring where I cached esm.sh resources and is occuring where I cleared the cache)

To reliable pin an esm version, using the same version for dependencies is essential.

**Comment by schlichtanders** [comment]
yes, I like that all can refer to the same version, however it is suboptimal that one always need to rely on the latest esm.sh version.  (as there is always only one "stable", i.e. the latest)

Currently the new version of "stable" completely broke my setup and I moved away from esm.sh as a consequence.

**Comment by ije** [comment]
sorry for the break change, let me make it clear, the stable means we locked the import path, you still can specify different version of these packages. eg:

```bash
https://esm.sh/react -> https://esm.sh/react@latest -> https://esm.sh/react@18.2.0 -> https://esm.sh/stable/react@18.2.0

https://esm.sh/react@17 -> https://esm.sh/react@17.0.2 -> https://esm.sh/stable/react@17.0.2
```

**Comment by schlichtanders** [comment]
yes, I fixed it to preact 10.6.6 like seen above, which was not enough to prevent the break

hence the issue, as it would be great to also pin these react/preact combos to a specific esm.sh version like `esm.sh/v66`

**Comment by ije** [comment]
in next build, build version like 'v135' or 'stable' will be removed. no _pin_ needed then.

**Comment by renhiyama** [comment]
Sounds like @ije is cooking! Let him cook 🔥
